.github/workflows/contracts.yml

```diff
@@ -81,4 +82,69 @@ jobs:
         run: npm ci
 
       - name: Lint
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+
+  migrate-and-publish-celo:
+    needs: [build-and-test, lint]
+    # FIXME: alfajores branch and pull request events are for test purposes only - revert once done
+    if: |
+      (startsWith(github.head_ref, 'releases/alfajores/')
+        || github.ref == 'refs/heads/master')
+        && (github.event_name == 'push'
+        || github.event_name == 'pull_request'
+        || github.event_name == 'workflow_dispatch')
+    environment: keep-test
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./solidity
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
```

Comment:
We could switch to using node matrix here - but this is something that I can change when working on migration job job for Ethereum chain, so you can leave it as it is...